icgcget/cli.py

```diff
@@ -18,20 +20,25 @@
 
 import logging
 import os
+import sys
 import json
 import click
 
-from commands.versions import versions_command
-from commands.reports import StatusScreenDispatcher
-from commands.download import DownloadDispatcher
-from commands.access_checks import AccessCheckDispatcher
-from commands.utils import compare_ids, config_parse, validate_ids, load_json
-from commands.configure import ConfigureDispatcher
+from icgcget.commands.versions import versions_command
+from icgcget.commands.reports import StatusScreenDispatcher
+from icgcget.commands.download import DownloadDispatcher
+from icgcget.commands.access_checks import AccessCheckDispatcher
+from icgcget.commands.utils import compare_ids, config_parse, validate_ids, load_json, validate_repos
+from icgcget.commands.configure import ConfigureDispatcher
 
 DEFAULT_CONFIG_FILE = os.path.join(click.get_app_dir('icgc-get', force_posix=True), 'config.yaml')
 REPOS = ['collaboratory', 'aws-virginia', 'ega', 'gdc', 'cghub', 'pdc']
```

Comment:
Basically something like this:

https://docs.python.org/3/library/enum.html#planet

But with `mass`, `radius` replaces with `code` and `name`. We should discuss this as you may need a converter for Click.